examples/deployment-with-init/deployment.tf

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
We normally use underscores to separate words in file and directory names. Will be good to rename this directory to `examples/deployment_with_init/`.